test/github-trigger

```diff
@@ -46,10 +47,15 @@ def trigger_pull(github, opts):
     ret = 0
     for context in contexts:
         status = statuses.get(context, { })
-        if status.get("state", all and "unknown" or "empty") not in ["empty", "error", "failure"]:
-            if not opts.force:
+        current_status = status.get("state", all and "unknown" or "empty")
+
+        if current_status not in ["empty", "error", "failure"]:
+            # allow changing if manual testing required, otherwise "pending" state indicates that testing is in progress
+            manual_testing = current_status == "pending" and status.get("description", None) == testinfra.NO_TESTING
```

Comment:
> What sets NO_TESTING?

testinfra defaults to that if priority is < 0: https://github.com/cockpit-project/cockpit/blob/master/test/common/testinfra.py#L517
notably this is the case for users not on the whitelist: https://github.com/cockpit-project/cockpit/blob/master/test/common/testinfra.py#L588